tests/e2e/080_vrf_test.go

```diff
@@ -193,11 +210,11 @@ func TestVRFsWithSameSubnets(t *testing.T) {
 	Expect(ctx.GetValueState(vppVrf1)).To(Equal(kvscheduler.ValueState_CONFIGURED))
 	Expect(ctx.GetValueState(vppVrf2)).To(Equal(kvscheduler.ValueState_CONFIGURED))
 
-	// try to ping in both VRFs
-	Expect(ctx.PingFromMs(msName, vrfVppIP, PingWithSourceInterface(vrf1Label+tapNameSuffix))).To(Succeed())
-	Expect(ctx.PingFromMs(msName, vrfVppIP, PingWithSourceInterface(vrf2Label+tapNameSuffix))).To(Succeed())
-
-	Expect(ctx.AgentInSync()).To(BeTrue())
+	// vrf mtu check
+	linuxVrf1Mtu := ctx.GetValue(linuxVrf1, kvs.SBView).(*linux_interfaces.Interface).Mtu
+	Expect(linuxVrf1Mtu == vrf.DefaultVrfDevMTU || linuxVrf1Mtu == vrf.DefaultVrfDevLegacyMTU).To(BeTrue())
```

Comment:
Use [Composing Matchers](https://onsi.github.io/gomega/#composing-matchers) for AND/OR operators so  GoMega can properly report which condition failed.